src/java/org/apache/poi/common/usermodel/fonts/FontGroup.java

```diff
@@ -42,10 +44,14 @@ Licensed to the Apache Software Foundation (ASF) under one or more
     COMPLEX_SCRIPT
     ;
 
-
     public static class FontGroupRange {
         private int len;
-        private FontGroup fontGroup;
+        private final FontGroup fontGroup;
+
+        public FontGroupRange(FontGroup fontGroup) {
```

Comment:
Adding a constructor here avoids needing to add `@Nullable` to the `fontGroup` field. I imagine this pattern would arise several times.

Reply:
I think it is safe to change this class now - because the 2 members are private so setting them is not really feasible from 3rd party code.

Reply:
https://github.com/apache/poi/commit/10652e33feb2ee164437ae265f98d8443a94aa19 -- I've removed the stray import in a 2nd commit